citrination_client/client.py

```diff
@@ -1,659 +1,15 @@
-        :param share: share the dataset with everyone on the site. Valid values are '1' or '0'.
-                      1 means share with everyone on the site. 0 means only the owner of the dataset
-                      can see the dataset.
-        :return: Response from the update data set request.
-        """
-        url = self._get_update_data_set_url(data_set_id)
-        data = {}
-        if name:
-            data["name"] = name
-        if description:
-            data["description"] = description
-        if share:
-            data["public"] = share
-        dataset = {"dataset": data}
-        return self._post_with_version_check(url, headers=self.headers, data=json.dumps(dataset))
-
-    def _get_update_data_set_url(self, data_set_id):
-        """
-        Helper method to generate the url for updating a data set.
-
-        :return: URL for creating a new data set.
-        """
-        return self.api_url+'/data_sets/'+str(data_set_id)+'/update'
-
-    def create_data_set_version(self, data_set_id):
-        """
-        Create a new data set version.
-
-        :param data_set_id: Id of the particular data set to upload to.
-        :return: Response from the create data set version request.
-        """
-        url = self._get_create_data_set_version_url(data_set_id)
-        return self._post_with_version_check(url, headers=self.headers, data=json.dumps({}))
-
-    def _get_create_data_set_version_url(self, data_set_id):
-        """
-        Helper method to generate the url for creating a new data set version.
-
-        :return: URL for creating new data set versions.
-        """
-        return self.api_url+'/data_sets/'+str(data_set_id)+'/create_dataset_version'
-
-    def _get_data_analysis_url(self, model_name):
-        return self.api_url + '/data_views/' + model_name + '/data_analysis'
-
-
-class QueryEncoder(json.JSONEncoder):
-    """
-    Class used to convert a query to json.
-    """
-
-    def default(self, obj):
-        """
-        Convert an object to a form ready to dump to json.
-
-        :param obj: Object being serialized. The type of this object must be one of the following: None; a single
-        object derived from the Pio class; or a list of objects, each derived from the Pio class.
-        :return: List of dictionaries, each representing a physical information object, ready to be serialized.
-        """
-        if obj is None:
-            return []
-        elif isinstance(obj, list):
-            return [i.as_dictionary() for i in obj]
-        elif isinstance(obj, dict):
-            return self._keys_to_camel_case(obj)
-        else:
-            return obj.as_dictionary()
 
-    def _keys_to_camel_case(self, obj):
-        """
-        Make a copy of a dictionary with all keys converted to camel case. This is just calls to_camel_case on
-        each of the keys in the dictionary and returns a new dictionary.
+    def __init__(self, api_key=None, site=None):
```

Comment:
Add suppress warnings and pass it through.

Reply:
👍